app/src/main/java/org/p2p/wallet/debug/torus/DebugTorusFragment.kt

```diff
@@ -58,6 +61,19 @@ class DebugTorusFragment : BaseFragment(R.layout.fragment_debug_torus) {
                 val newVerifier = verifierEditText.text.toString()
                 updateEnvironmentAndRestart(newUrl = newUrl, newVerifier = newVerifier)
             }
+
+            val signUpDetails = signUpDetailsStorage.getLastSignUpUserDetails()
+            val hasShare = signUpDetails?.signUpDetails?.deviceShare != null
+            shareTextView.text = if (hasShare) {
+                signUpDetails?.signUpDetails?.deviceShare.toString()
+            } else "No device share"
```

Comment:
```suggestion
            shareTextView.text = if (hasShare) {
                signUpDetails?.signUpDetails?.deviceShare.toString()
            } else { 
              "No device share"
            }
```

also - deviceShare.toString() will represent the whole object in String representation. do you need a string share value here or a whole object with verifiers and stuff?

Reply:
i think it will be cool to check some data if QA needs